be/test/storage/lake/partial_update_test.cpp

```diff
@@ -581,6 +581,73 @@ TEST_P(PartialUpdateTest, test_write_with_index_reload) {
     }
 }
 
+TEST_P(PartialUpdateTest, test_partial_update_publish_retry) {
+    auto chunk0 = generate_data(kChunkSize, 0, false, 3);
+    auto chunk1 = generate_data(kChunkSize, 0, true, 3);
+    auto indexes = std::vector<uint32_t>(kChunkSize);
+    for (int i = 0; i < kChunkSize; i++) {
+        indexes[i] = i;
+    }
+
+    auto version = 1;
+    auto tablet_id = _tablet_metadata->id();
+    // normal write
```

Comment:
Are these extra normal writes necessary for the test?

Reply:
Yes, We normal write first, and then partial update write.

Reply:
If so, can it be written only once to reduce the test run time?

Reply:
Yes, of course. Done.